ld_gain: 1.0
min_ld: 0.5
car_wheel_base: 0.2
controller_freq: 50
max_speed: 0.6
map_frame: "map"
base_frame: "base_link"